README.md

```diff
@@ -0,0 +1,40 @@
+# Hyperfoil.io Website Based on Jekyll
+
+## Getting Started
+
+These instructions will get you a copy of the Hyperfoil.io website up and running on your local machine for development and testing purposes.
+
+### Installation
+[Jekyll static site generator docs](https://jekyllrb.com/docs/).
+
+1. Install a full [Ruby development environment](https://jekyllrb.com/docs/installation/)
+2. Install [bundler](https://jekyllrb.com/docs/ruby-101/#bundler)  [gems](https://jekyllrb.com/docs/ruby-101/#gems) 
+  
+        gem install bundler
+
+3. Fork the [project repository](https://github.com/Hyperfoil/hyperfoil.github.io), then clone your fork.
+  
+        git clone git@github.com:YOUR_USER_NAME/hyperfoil.github.io.git
+
+4. Change into the project directory:
+  
+        cd hyperfoil.github.io
+
+5. Use bundler to fetch all required gems in their respective versions
+
+        bundle install
+
+6. Build the site and make it available on a local server
+  
+        bundle exec jekyll serve
+        
+7. Now browse to http://localhost:4000
+
+> If you encounter any unexpected errors during the above, please refer to the [troubleshooting](https://jekyllrb.com/docs/troubleshooting/#configuration-problems) page or the [requirements](https://jekyllrb.com/docs/installation/#requirements) page, as you might be missing development headers or other prerequisites.
+
+**For more regarding the use of Jekyll, please refer to the [Jekyll Step by Step Tutorial](https://jekyllrb.com/docs/step-by-step/01-setup/).**
+
+
+## License
+
+This website is licensed under the [Creative Commons Attribution 3.0](https://creativecommons.org/licenses/by/3.0/).
```

Comment:
@rvansa I wanted to ask about this. There is no License for the website, and this is the license that the Quarkus uses, IDK if you have thought about licensing for the website. TBH I am not sure what would be appropriate?

Reply:
CC-BY is fine for the content I wrote - I only hope it does not conflict with the fact that parts of the webpage can be either generated based on ASL sources or otherwise related (e.g. there's a report template). Also the I am not sure if we can decide on licensing of logos - in this case I would consider CC-BY-ND.